packages/file-tree-next/src/browser/services/file-tree-event.service.ts

```
@@ -0,0 +1,18 @@
+import { Injectable } from '@opensumi/di';
+import { Emitter, Event, FilesChangeEvent, OnEvent, WithEventBus } from '@opensumi/ide-core-browser';
+
+@Injectable()
+export class FileTreeEventService extends WithEventBus {
+  private _onDidChange: Emitter<null> = new Emitter();
+
+  public onDidChange: Event<null> = this._onDidChange.event;
+
+  constructor() {
+    super();
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)